### PR TITLE
D4di manuscript re org

### PR DESCRIPTION
**Pull request by mariaruth**
I updated manuscript.tex to reflect the changes in chapter organization and titles, as it was getting confusing with the old/new numbering. @luizaandrade i assumed the new  chapter 5 file will be data-processing.tex, please update if not.

### CODE REVIEW

File: manuscript.tex [hunk 1]
```diff
@@ -28,60 +28,54 @@ \chapter{Introduction: Data for development impact} % The asterisk leaves out th
...
-\input{chapters/sampling-randomization-power.tex}
+\input{chapters/data-collection.tex}
 
 %----------------------------------------------------------------------------------------
 %	CHAPTER 5
 %----------------------------------------------------------------------------------------
 
-\chapter{Chapter 5: Acquiring development data}
-\label{ch:5}
+\chapter{Chapter 6: Processing research data}
```
Comment (luizaandrade) [inline manuscript.tex:75]:
```suggestion
\chapter{Chapter 5: Processing research data}
```

File: manuscript.tex [hunk 2]
```diff
@@ -28,60 +28,54 @@ \chapter{Introduction: Data for development impact} % The asterisk leaves out th
...
+\label{ch:6}
```
Comment (luizaandrade) [inline manuscript.tex:76]:
```suggestion
\label{ch:5}
```

**Comment by luizaandrade** [comment]
Can I go ahead an merge this? @bbdaniels @kbjarkefur

**Comment by bbdaniels** [comment]
Can we use this PR to get all the chapter names and file names agreed first? I don’t think it’s done yet.

**Comment by kbjarkefur** [comment]
@luizaandrade @bbdaniels @mariaruth Lets discuss how to merge all these PR. Since so many branches have deleted sections in one file that another branch has completely rewritten, we will get ostensibly terrible merge conflicts. 

However i think they can be solved easily, but Id like to make a replica of this repo and test first. I can do thah after our meeting tomorrow.

**Comment by kbjarkefur** [comment]
I'd propose to delete this PR and branch and do this one we have solved the file names etc in the develop branch. Some branches worked in Luiza's new file names, and some in the old. And some branches updated the `manuscript.tex` and some didn't. I think re-doing this is less work than updating this. Sorry.

**Comment by bbdaniels** [comment]
Whatever you think is easiest I’ll support!

**Comment by kbjarkefur** [comment]
This has now been done in a different branch

**Pull request closed without merging**